packages/flutter_tools/templates/create/pubspec.yaml.tmpl

```diff
@@ -1,6 +1,13 @@
 name: {{projectName}}
 description: {{description}}
 
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by a build number separated by a +.
+# A complete version number looks like 1.2.43+177
+# Flutter will use this number to set the version and build number
+# for Android and iOS builds.
```

Comment:
Please state the meaning of the three numbers (e.g. referring to semver.org).
Please terminate full sentences with a period.
A convention of this file seems to be to start documentation with `The following`.
Another convention seems to be to make example syntax stand out by putting it on its own line

For example:
```yaml
# The following defines the version and build number for your application. A version string
# has three blahblah semver.org blahblah followed by a + blahblah. Example:
# version: 1.2.43+177
version: 1.0.0+1
```

Reply:
done